### DISCUSSION TITLE
PSA: CLM-FATES is now available as a Galaxy tool.

### DISCUSSION THREAD

**Discussion by fubar2** [NGEET/fates#1013] [Announcements]
(Apologies for this busybody bandwidth - let me know and I'll close it if this is not of interest)

Is it worth adding something about https://climate.usegalaxy.eu to https://github.com/NGEET/fates/wiki/Choosing-a-Version-of-FATES ? Ah - I see https://github.com/NGEET/docker-fates-tutorial/issues/30 so this is no surprise...

I can submit a PR if nobody objects?

User overview looks like this:

https://climate.usegalaxy.eu offers free access to CLM-FATES, with a GUI workflow editor for shareable, repeatable experiments, with the CLM, Panoply viewer and interactive Pangeo Jupyter environments. Ideal for FAIR global science, accessed from any internet browser. Best place to start if this sounds interesting is to make your own private account at https://climate.usegalaxy.eu, then use the teaching material at https://training.galaxyproject.org/training-material/topics/climate/ (and the Galaxy 101 intro unless you have used Galaxy before). 

Disclosure: This is not my work, and I know nothing about FATES but I am involved with the Galaxy project and community.

**Comment by rosiealice** [comment]
Hi Ross,

Thanks for this. It would be great to have a reference to the Galaxy portal somewhere on our site. It's still true, however, that scientifically, FATES must be run through a host land model. In this case, CLM, which provides the boundary conditions for running FATES and all the surface hydrology, energy balance, soil biogeochemistry and so on. The Galaxy portal is in effect a way of running the CLM/CTSM code with a FATES compset (although in principle I think it could run any complet?)  

I think what is needed is probably a new paragraph on the possibility of using Galaxy to drive CLM-FATES online? We can iterate on this if you'd like...

Thanks again!

**Comment by fubar2** [comment]
Hi @rosiealice! Thanks for the response - point taken - so this is CLM-FATES as I finally figured out. 
As I said I have zero experience with the area - just lucky enough to see some smart collaborators doing what appears to be very interesting work using Galaxy. 
It might be best if people who understand this stuff do the writing.
If the paragraph I wrote can be recycled into something that makes sense for your visitors and users, that would be excellent! If I can be of any help, please let me know.
